### ISSUE TITLE
Code doesn't work after formatting

### ISSUE THREAD

**Issue by mozesa** [prettier/plugin-php#1959]
**Prettier 2.0.0**

**PHP Plugin 0.18.4**

```json
"prettier": {
  "phpVersion": "8.0",
  "useTabs": true,
  "singleQuote": true
}
```

**Input:**

```php
$measurement  = (fn($m): Measurement => $m)(Measurement::query()->find($id, ['time', 'raw_gyro']));
```
![image](https://user-images.githubusercontent.com/49973426/160775168-588bc476-68c0-4aba-b298-d5eb78ac08cc.png)

**Output:**

```php
$measurement = fn($m): Measurement => $m(
	Measurement::query()->find($id, ['time', 'raw_gyro']),
);
```
![image](https://user-images.githubusercontent.com/49973426/160775222-367871c9-f572-4fad-affd-7c368dfcf610.png)

**Expected behavior:**
Won't broke the code.

**Comment by cseufert** [comment]
Yes, it appears the closure is being unwrapped at some point.

Simpler example
```
$a = (fn() => 0)(1);
```
becomes
```
$a = fn() => 0(1);
```

**Comment by snake-py** [comment]
Seeing this bug as well

```json
{
    "phpVersion": "8.1",
    "printWidth": 80,
    "tabWidth": 4,
    "useTabs": false,
    "singleQuote": false,
    "trailingCommaPHP": true,
    "braceStyle": "psr-2",
    "requirePragma": false,
    "insertPragma": false
}
```

**Comment by czosel** [comment]
Released in `0.19.1` :tada:
